certified-connectors/Nodefusion Portal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,166 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Nodefusion Portal",
+    "description": "A Nodefusion Portal Custome Connector is used for various actions, such as opening support case, or subscription activation",
+    "contact": {
+      "name": "Nodefusion",
+      "url": "https://www.nodefusion.com",
+      "email": "nodefusion@nodefusion.com"
+    },
+    "license": {
+      "name": "MIT",
+      "url": "http://opensource.org/licenses/MIT"
+    },
+    "version": "5.1.0.4732"
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.nodefusion.com"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.nodefusion.com/terms"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "IT Operations;Website"
+    }
+  ],
+  "host": "3proportal-functions.azurewebsites.net",
+  "basePath": "/api",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/organizations/{organizationId}/Profile": {
+      "get": {
+        "tags": [
+          "Organization",
+          "Profile"
+        ],
+        "summary": "Get Organization Profile",
+        "description": "This function returns the data about the requested organization if you have permissions",
+        "operationId": "GetOrgProfile",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "organizationId",
+            "x-ms-summary": "organizationId",
+            "x-ms-url-encoding": "single",
+            "description": "This parameter is a Guid of your Organization",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Payload of OrganizationProfileResponse",
+            "schema": {
+              "$ref": "#/definitions/organizationProfileResponse"
+            },
+            "x-ms-summary": "The response"
+          }
+        },
+        "security": [

```

Comment:
@hkusulja Please let us know the reason to add this property? You can find the security properties in the apiProperties.json file which would apply to the operations of the connector.

Reply:
@sriyen-msft it is generated like that by swagger generator in our function